client/migrations/0001_initial.py

```diff
@@ -17,17 +22,22 @@ class Migration(migrations.Migration):
                                                max_length=32,
                                                primary_key=True,
                                                serialize=False,
-                                               validators=[django.core.validators.MinLengthValidator(1)])),
+                                               validators=[django.core.validators.MinLengthValidator(6)])),
                 ('first_name', models.CharField(help_text='First Name',
                                                 max_length=32,
-                                                null=True)),
+                                                validators=[django.core.validators.MinLengthValidator(1)])),
```

Comment:
wondering how there wasnt validators before uve added....

Reply:
All good! you guys just didnt update the migrations with new ones

Reply:
> > ignore (cant delete).
> 
> what

made a comment, saw it was irrelevant, couldnt delete the comment so ignore it